thinkhazard/views/pdf.py

```diff
@@ -151,7 +151,7 @@ def create_pdf_report(request):
         tempfile.gettempdir(), object_name
     )
 
-    s3_helper = _create_s3_helper(request)
+    s3_helper = S3Helper(request.registry.settings)
```

Comment:
Have you checked performance cost when hitting real S3 backend at each request?

Plus, if you create the helper here, you'd better store in the request for reuse everywhere instead of creating other helpers in the code.

Reply:
Or add this in main:
```
config.add_request_method(
    "s3_helper",
    lambda r: r.S3Helper(request.registry.settings),
    property=True,
    reify=True,
)
```
After what you can use ```request.s3_helper```

Reply:
Good points, thanks. I adapted as you proposed. 

Indeed the performance could be better (turning around 300ms for the remote s3 calls) but the s3 seems to be the most persistent solution in the new infra for now.

Reply:
300ms is reponse time when running the webapp locally, but actually home page request take 100ms on production server.

Reply:
We may use a table in public schema of public database ?

Reply:
Or it may be possible to completely restore the public database in a single transaction.
It such case, no need to disable public UI.

Reply:
Ok after discussion we take out maintenance mode for now due to short downtime and just take the refactoring part from this PR